kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -380,15 +380,19 @@ public boolean autoApproveProcurementCardDocuments() {
                 Timestamp docCreateDate = new Timestamp( pcardDocument.getDocumentHeader().getWorkflowDocument().getDateCreated().getMillis() );
                 if (KfsDateUtils.getDifferenceInDays(docCreateDate, currentDate) > autoApproveNumberDays) {
                     // update document description to reflect the auto approval
-                    pcardDocument.getDocumentHeader().setDocumentDescription("Auto Approved On " + dateTimeService.toDateTimeString(currentDate) + ".");
+                    if (pcardDocument.getDocumentHeader().getDocumentDescription().startsWith("FY ")) {
```

Comment:
If description starting with FY - PCDO description field display is replaced during procurementCardDocumentJob batch process if document is a fiscal year end document or an auto-approved document, or both.